UnitTests/MockWebAuthenticationSession.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+import AuthenticationServices
+@testable import PopupBridge
+
+class MockWebAuthenticationSession: WebAuthenticationSession {
```

Comment:
This is also another case for breaking out our ASWeb wrapper from BTCore into it's own module - PopupBridge could consume it too. 🏈